core/src/main/resources/hudson/util/HudsonIsRestarting/index_da.properties

```diff
@@ -20,5 +20,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Your\ browser\ will\ reload\ automatically\ when\ Jenkins\ is\ ready.=Din browser vil genindlæse siden automatisk når Jenkins er klar.
-Please\ wait\ while\ Jenkins\ is\ restarting=Vent venligst imens Jenkins genstarter
```

Comment:
Was this intentional? I don't speak the language but Google Translate says it's now "Please wait Jenkins restarts" which seems not great translated back.